src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -1908,7 +1909,7 @@ public function set_attribute( $name, $value ) {
 			 *    Result: <div id="new"/>
 			 */
 			$existing_attribute             = $this->attributes[ $comparable_name ];
-			$this->lexical_updates[ $name ] = new WP_HTML_Text_Replacement(
+			$this->lexical_updates[ $comparable_name ] = new WP_HTML_Text_Replacement(
```

Comment:
Indentation of `=` signs seems no longer aligned with respect to the previous line. Kinda surprised our WPCS CI job didn't flag this 🤔

Reply:
I just pushed out c21f8fa622 which adjusts the alignment and I'm curious. so far, when things look wrong but they pass the linter it's because the linter prefers the less-legible format 😆